cranelift/codegen/src/prelude_lower.isle

```diff
@@ -1001,6 +1001,10 @@
 (decl abi_sized_stack_ret_space (Sig) i64)
 (extern constructor abi_sized_stack_ret_space abi_sized_stack_ret_space)
 
+;; Incoming return area pointer.
+(decl abi_ret_area_ptr () Reg)
+(extern constructor abi_ret_area_ptr abi_ret_area_ptr)
```

Comment:
And then this external constructor could be called `unwrap_abi_ret_area_ptr` or some such.